math_.py

```diff
@@ -3,6 +3,13 @@
 def sum_(a: int, b: int) -> int:
     return a+b
 
-def list_sum(a: list, b: list) -> list:
+def list_sum(a: List, b: List) -> List:
     assert len(a)==len(b), "len(a) != len(b)"
     return [i+j for i, j in zip(a,b)]
+
+def substract(a: int, b: int) -> int:
+    return a-b
+
```

Comment:
This assertion is duplicated from the `list_sum` function.  It might be worth extracting this into a separate helper function to avoid redundancy.